### PR TITLE
Fix(cohort-date-range-custom-dates): added ability to make custom dates option active, updated storybook

### PR DESCRIPTION
**Pull request by ZhmenZH**

### CODE REVIEW

File: stories/ui-cohort-date-range/cohort-date-range.stories.ts [hunk 1]
```diff
@@ -95,6 +94,31 @@ const COHORTS_ACTIVE: TsDateCohort[] = [
...
+  },
+  {
+    display: 'Last full month',
+    range: {
+      start: startOfDay(subMonths(startOfMonth(DATES.end), 1)),
+      end: endOfDay(subDays(startOfMonth(DATES.end), 1)),
+    },
+  },
+  {
+    display: 'Custom Dates',
```
Comment (belinef) [inline stories/ui-cohort-date-range/cohort-date-range.stories.ts]: looks like should be `TS_CUSTOM_DATES_DESCRIPTION`, no ?

**Review by belinef — CHANGES_REQUESTED** [review]

File: stories/ui-cohort-date-range/cohort-date-range.stories.ts (hunk 1 above)
Comment (ZhmenZH) [reply stories/ui-cohort-date-range/cohort-date-range.stories.ts]: no, stories and libs it is different directories

Comment (belinef) [reply stories/ui-cohort-date-range/cohort-date-range.stories.ts]: i see imports from library 

consumers should e able to import it

Comment (ZhmenZH) [reply stories/ui-cohort-date-range/cohort-date-range.stories.ts]: done

**Review by belinef — APPROVED** [review]

**Review by lexasq — APPROVED** [review]

**Review by ChrisKittredge — APPROVED** [review]
Looks good to me

**Pull request merged**